### DISCUSSION TITLE
Videos por si quieren aprender mas html

### DISCUSSION THREAD

**Discussion by DiegoDominguez3132** [Launch-X-Latam/MisionFrontEnd#170] [General]
Holaa, por si quieren investigar mas sobre html les recomiendo dos canales que tienen un curso sobre esto, uno es Soy Dalto,aca les dejo el video https://www.youtube.com/watch?v=kN1XP-Bef7w&t=7849s y el otro por si tienen mas tiempo es Dorian desings https://www.youtube.com/watch?v=vz4z0RLcAyk&t=119s

**Comment by doguedogue** [comment]
gracias man, también te recomiendo a https://fernando-herrera.com/ es genial, yo he aprendido mucho con él.
